### DISCUSSION TITLE
Serverpod Raspberry Pi Deployment

### DISCUSSION THREAD

**Discussion by MrFahrenheit-55** [serverpod/serverpod#2401] [General]
Good day everyone in the awesome community!

I want to move my Serverpod Developer instance to a raspberry pi for ease of testing between multiple users.

1) Has anyone ever deployed Serverpod on a Raspberry Pi using Docker or similar?
2) How do I migrate the entire dev database with data to the new server?

I really appreciate all the help.

Regards

**Comment by Isakdl** [comment]
Hey @MrFahrenheit-55 

I have not tried hosting on a raspberry pi but it should be possible I think. The Docker container supports both x86 and ARM, would be an interesting exercise to try!

How to move data see this excellent StackOverflow thread: https://stackoverflow.com/questions/1237725/copying-postgresql-database-to-another-server

**Comment by aldychris** [comment]
Hi @MrFahrenheit-55  have you tried to host Serverpod Developer instance on raspberry?

**Reply by MrFahrenheit-55** [reply]
Hi @aldychris, I did try it a while back and got it to run for a new instance. I stopped when I ran into some errors when I tried to migrate my database from my machine to the Pi. I have not looked at it since.
